contrib_bots/lib/github_detail.py

```diff
@@ -0,0 +1,135 @@
+import re
+import os
+import sys
+import six.moves.configparser
+
+import requests
+
+from typing import Text, Dict
+
+
+class GithubHandler(object):
+    '''
+    This plugin provides details on github issues and pull requests when they're refrenced in the chat.
+    '''
+
+    config_path = os.path.expanduser('~/.contrib_bots/github_detail.ini')
+    owner = Text
```

Comment:
Delete this. It's not required. Same for `repo = Text`. As a side note, you are assigning a class (Text) to `repo`. This is a logical bug.